comments.php

```diff
@@ -29,7 +29,8 @@
 
 <div class="comments wrap__item">
 
-	<amp-live-list id="amp-live-comments-list-<?php the_ID(); ?>" class="live-list" layout="container" data-poll-interval="<?php echo esc_attr( AMPCONF_LIVE_LIST_POLL_INTERVAL ); ?>" data-max-items-per-page="<?php echo esc_attr( get_option( 'page_comments' ) ? get_option( 'comments_per_page' ) : 10000 ); ?>">
+	<?php $sort_attr = ( 'asc' === get_option( 'comment_order' ) ) ? ' sort="ascending" ' : ''; ?>
```

Comment:
Maybe this could be `'sort="ascending"'` instead of `' sort="ascending" '`. When this is echoed on the next line, there are already empty spaces before and after.